esp/bindings/http/client/httpclient.cpp

```diff
@@ -1001,3 +1008,7 @@ IHttpClientContext* createHttpClientContext(IPropertyTree* config)
     return new CHttpClientContext(config);
 }
 
+MODULE_EXIT()
```

Comment:
Trivial, but the convention seems to be to put the MODULE_EXIT() at the top right after the MODULE_INIT().  It's easy to keep them in sync that way.

Reply:
Change made as suggested.